src/AddIns/Uno.UI.Lottie/LottieVisualSource.wasm.cs

```diff
@@ -53,13 +53,25 @@ public void Update(AnimatedVisualPlayer player)
 			_isPlaying = player.AutoPlay;
 		}
 
-		private void OnStateChanged(object sender, EventArgs e)
+		private void OnStateChanged(object sender, HtmlCustomEventArgs e)
 		{
-			var r = WebAssemblyRuntime.InvokeJS("Uno.UI.Lottie.getAnimationState(" + _player.HtmlId + ");");
+			//var r = WebAssemblyRuntime.InvokeJS("Uno.UI.Lottie.getAnimationState(" + _player.HtmlId + ");");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436651)